index.html

```diff
@@ -25,150 +25,225 @@
     <!-- Custom css -->
     <link href="/styles/style.css" rel="stylesheet">
 
-  </head>
-  <body>
 
-    <!-- Following Menu -->
-    <div class="ui large top fixed hidden menu">
-      <div class="ui container">
-        <a class="active item">Home</a>
-        <a class="item">Work</a>
-        <a class="item">Company</a>
-        <a class="item">Careers</a>
-      </div>
-    </div>
+    <style type="text/css">
+
+      .hidden.menu {
```

Comment:
If any of these aren't in the [custom stylesheet](https://github.com/jakeorr/oktabletop/blob/master/public/styles/style.css) can you move them in there? Let's leave all our styles in there, this style block can be removed.